src/components/DragCaptureZone/DragCaptureZone.jsx

```diff
@@ -65,24 +72,40 @@ const DragCaptureZone = createClass({
 	render() {
 		return (
 			<div
-				{...this.props}
+				{...omitProps(this.props, DragCaptureZone)}
 				className={cx('&', this.props.className)}
 				key='DragCaptureZone'
 				onMouseDown={this.handleDragStart}
+				ref={(ref) => {this.elementRef = ref;}}
 			/>
 		);
 	},
 
+	componentDidMount() {
+		//add event listeners directly on the DOM elemnt to allow preventDefault
+		//calls which are not honored due to react's event delegation
```

Comment:
this is the Chrome only bug you were talking about right? Link to some related reading might be useful as part of the comment or something.